src/Workspaces/Core/Portable/Remote/RemoteHostClient.cs

```diff
@@ -128,13 +131,15 @@ public void Dispose()
             }
 
 #if DEBUG
+            private string _creationCallStack;
```

Comment:
readonly?

Reply:
sure.

Reply:
Why only debug?  still seems useful in release.